docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -3,47 +3,47 @@
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant ":BookFaceParser" as BookFaceParser LOGIC_COLOR
+participant ":PrimaryParser" as PrimaryParser LOGIC_COLOR
 participant ":DeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:DeleteCommand" as DeleteCommand LOGIC_COLOR
+participant ":DeleteUserCommand" as DeleteUserCommand LOGIC_COLOR
```

Comment:
`DeleteUserCommand` is not used in this sequence diagram.

Reply:
Yep, currently changing this to reflect the current procedure